### ISSUE TITLE
Reordering activities should be possible by drag and drop

### ISSUE THREAD

**Issue by opcoach** [bookingactivities/booking-activities#43]
In the calendar editor, it is possible to move activities but they are not reordered. 

In the attached screenshot, I created a new activity with the orange color, and I would like to move it with the others. Basically, I use colors for each different trainer, and it would be handy to reorder them.

<img width="506" alt="CalendarActivitiesReorder" src="https://user-images.githubusercontent.com/1716878/70951310-5c437f80-2063-11ea-8d56-c89cc29973db.png">

**Comment by yoancutillas** [comment]
Thank you for your feedback,
This cannot be done easily, but I note your request.

For now, you can set the order by color by code:
```
function my_theme_order_activities_in_calendar_editor() {
	return array( 'color', 'title', 'id' );
}
add_filter( 'bookacti_activities_list_order_by', 'my_theme_order_activities_in_calendar_editor' );
```

**Comment by opcoach** [comment]
Cool that works well.  I put it in my custom plugin extension. 

This bug could be fixed also with a toolbar containing the different sorter : by color, by name, by nb of events, etc...

**Comment by pikurasa** [comment]
I have also thought that being able to reorder would be useful, so +1 from me.

**Comment by yoancutillas** [comment]
Released in Booking Activities 1.11
